src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -0,0 +1,778 @@
+    </Compile>
+    <Compile Include="..\service\ServiceUntypedParse.fs">
+      <Link>Service/ServiceUntypedParse.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceAssemblyContent.fsi">
+      <Link>Service/ServiceAssemblyContent.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceAssemblyContent.fs">
+      <Link>Service/ServiceAssemblyContent.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceXmlDocParser.fsi">
+      <Link>Service/ServiceXmlDocParser.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceXmlDocParser.fs">
+      <Link>Service/ServiceXmlDocParser.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ExternalSymbol.fsi">
+      <Link>Service/ExternalSymbol.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ExternalSymbol.fs">
+      <Link>Service/ExternalSymbol.fs</Link>
+    </Compile>
+    <Compile Include="..\service\QuickParse.fsi">
+      <Link>Service/QuickParse.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\QuickParse.fs">
+      <Link>Service/QuickParse.fs</Link>
+    </Compile>
+    <Compile Include="..\..\fsharp\service\FSharpCheckerResults.fsi">
+      <Link>Service/FSharpCheckerResults.fsi</Link>
+    </Compile>
+    <Compile Include="..\..\fsharp\service\FSharpCheckerResults.fs">
+      <Link>Service/FSharpCheckerResults.fs</Link>
+    </Compile>
+    <Compile Include="..\service\service.fsi">
+      <Link>Service/service.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\service.fs">
+      <Link>Service/service.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceInterfaceStubGenerator.fsi">
+      <Link>Service/ServiceInterfaceStubGenerator.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceInterfaceStubGenerator.fs">
+      <Link>Service/ServiceInterfaceStubGenerator.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceStructure.fsi">
+      <Link>Service/ServiceStructure.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceStructure.fs">
+      <Link>Service/ServiceStructure.fs</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceAnalysis.fsi">
+      <Link>Service/ServiceAnalysis.fsi</Link>
+    </Compile>
+    <Compile Include="..\service\ServiceAnalysis.fs">
+      <Link>Service/ServiceAnalysis.fs</Link>
+    </Compile>
+    <Compile Include="..\fsi\fsi.fsi">
+      <Link>InteractiveSession/fsi.fsi</Link>
+    </Compile>
+    <Compile Include="..\fsi\fsi.fs">
+      <Link>InteractiveSession/fsi.fs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <!--
+    Some users have found it easier to work on FSharp.CompilerService when referencing the nuget package rather than the fsharp.core project
+    set FSHARPCORE_USE_PACKAGE to true to use package
+  -->
+    <FSHARPCORE_USE_PACKAGE>false</FSHARPCORE_USE_PACKAGE>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(FSHARPCORE_USE_PACKAGE)' != 'true'">
```

Comment:
As discussed in comments, I think it's best to change both FSharp.Compiler.Private and FSharp.Compiler.Service to compile against a stable published FSharp.Core and drop their refrence to FSharp.Core.fsproj.  At least FSharp.Compiler.Service should continue to be compiled this way.